src/components/Banner/Delegation/DelegationBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import './DelegationBanner.css'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import React, { useEffect, useState } from 'react'
+
+import Delegate from '../../Icon/Delegate'
+import Banner from '../Banner'
+
+type DelegationBannerProps = {
+  isVisible: boolean
+}
+
+const HIDE_DELEGATE_BANNER_KEY = 'org.decentraland.governance.delegate_banner.hide'
+
+function DelegationBanner({ isVisible }: DelegationBannerProps) {
+  const t = useFormatMessage()
+  const [showBanner, setShowBanner] = useState(false)
```

Comment:
I think showBanner can be just a variable without a useState.

`const showBanner = localStorage.getItem(HIDE_DELEGATE_BANNER_KEY) != 'true'`

Reply:
localStorage must be inside a useEffect
![image](https://user-images.githubusercontent.com/45410089/163451145-cee18f0c-6072-40eb-a81e-e3726a267f98.png)